index.html

```diff
@@ -514,43 +523,66 @@
         </dt>
       </dl>
       <p>
-        A <code>PermissionStatus</code> instance has an associated
-        <var>permission</var> that is a <a>PermissionDescriptor</a>.
+        <code>PermissionStatus</code> instances are created with the following
```

Comment:
I don't care strongly at the moment. Hopefully going forward IDL will provide formalism around this so you can define the layout of an object, including its internal slots, upfront.